ink/crates/phat_rollup_anchor_ink/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "phat_rollup_anchor_ink"
+version = "0.0.1"
+authors = ["GuiGou"]
+edition = "2021"
+
+[dependencies]
+ink = { version = "4.2.0", default-features = false }
```

Comment:
Should this be `ink = { version = "4.2", default-features = false }`?

Reply:
From my understanding, there is no difference between `version = "4.2.0" `and `version = "4.2"` but I don't know yet the good pratices in rust (I develop in a rust for less than a year). If it's better to remove the last digit when it is useless,, I will do the update :)

Reply:
Both "4.2.0" and "4.2" looks good to me, no difference.